src/main/java/calculator/strategy/FixedTextDelimiter.java

```diff
@@ -0,0 +1,37 @@
+package calculator.strategy;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class FixedTextDelimiter implements TextDelimiter {
+    private static final String DEFAULT_FIXED_DELIMITER = ",|:";
+    private final String delimiter;
+
+    public FixedTextDelimiter() {
+        this(DEFAULT_FIXED_DELIMITER);
+    }
+
+    public FixedTextDelimiter(final String delimiter) {
+        this.delimiter = delimiter;
+    }
+
+    @Override
+    public boolean isSupport(final String text) {
+        final Pattern pattern = Pattern.compile(delimiter);
+        final Matcher matcher = pattern.matcher(text);
+        return matcher.find();
+    }
+
+    @Override
+    public List<String> split(final String text) {
+        final Pattern pattern = Pattern.compile(text);
```

Comment:
```suggestion
        final Pattern pattern = Pattern.compile(delimiter);
```

요렇게 되어야 할 것 같은데 의도된 동작이실까요?

Reply:
`테스트가 왜 통과되었지..?` 라는 생각이 드네요.
이 부분 코드 수정하고 테스트 보강하여 잡아보도록 하겠습니다!